ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -8264,3 +8264,15 @@ DTPM:
   - detergent_pump_run_command
   implements:
   - MONITORING
+
+#TW-NTC-TPKE
```

Comment:
Please remove any comments referencing a specific building

Reply:
Hi, I have removed the comments in the new commit.